docs/src/modules/components/AppWrapper.js

```diff
@@ -128,27 +158,7 @@ async function registerServiceWorker() {
 }
 
 function AppWrapper(props) {
```

Comment:
```suggestion
export default function AppWrapper(props) {
```

Reply:
Can't use that here because of `prop-types`

Reply:
Okay get it.

Reply:
We have some code that does it:
https://github.com/mui-org/material-ui/blob/e499a0e93f04628724332d42c8156232786cc569/docs/src/modules/components/AppTheme.js#L9-L19

What issue does it create?

Reply:
Yeah right we can actually mutate exports.